### PR TITLE
Added description for softmax and sigmoid

### PR DESCRIPTION
**Pull request by aweers**
It's my first contribution so I'm not sure whether everything is right the way i did it. Also I am not a native english speaker so maybe there are some spelling errors.

### CODE REVIEW

File: tensorflow/python/keras/activations.py [hunk 1]
```diff
@@ -42,14 +42,18 @@
 
 @keras_export('keras.activations.softmax')
 def softmax(x, axis=-1):
-  """Softmax activation function.
+  """The softmax activation function transforms the outputs so that all values are in 
+  range (0, 1) and sum to 1. It is is often used as activation of the last layer of a
```
Comment (kyscg) [inline tensorflow/python/keras/activations.py]: Maybe change it to:
`It is often used as the activation for the last layer`

File: tensorflow/python/keras/activations.py [hunk 2]
```diff
@@ -42,14 +42,18 @@
 
 @keras_export('keras.activations.softmax')
 def softmax(x, axis=-1):
-  """Softmax activation function.
+  """The softmax activation function transforms the outputs so that all values are in 
+  range (0, 1) and sum to 1. It is is often used as activation of the last layer of a 
+  classification network because the result could be interpreted as probability
```
Comment (kyscg) [inline tensorflow/python/keras/activations.py]: And this:
`because the result could be interpreted as a probability`

File: tensorflow/python/keras/activations.py [hunk 3]
```diff
@@ -42,14 +42,18 @@
 
 @keras_export('keras.activations.softmax')
 def softmax(x, axis=-1):
-  """Softmax activation function.
+  """The softmax activation function transforms the outputs so that all values are in 
+  range (0, 1) and sum to 1. It is is often used as activation of the last layer of a 
+  classification network because the result could be interpreted as probability 
+  distribution. To calculate the softmax all inputs `x` become exp(x) divided by the sum
```
Comment (kyscg) [inline tensorflow/python/keras/activations.py]: Here:
`To calculate the softmax, all inputs...`

File: tensorflow/python/keras/activations.py [hunk 4]
```diff
@@ -170,6 +174,17 @@ def tanh(x):
 
 @keras_export('keras.activations.sigmoid')
 def sigmoid(x):
+  """Sigmoid.
+  Applies the sigmoid activation function. The sigmoid function is defined as 
+  1 divided by (1 + exp(-x)). It's curve is like an "S" and is like a smoothed 
+  version of the Heaviside function. For small values (<-5) the sigmoid returns
```
Comment (kyscg) [inline tensorflow/python/keras/activations.py]: Maybe put "Unit Step Function" in parenthesis for the Heaviside, the whole point after all is simplicity.

File: tensorflow/python/keras/activations.py [hunk 5]
```diff
@@ -170,6 +174,17 @@ def tanh(x):
 
 @keras_export('keras.activations.sigmoid')
 def sigmoid(x):
+  """Sigmoid.
+  Applies the sigmoid activation function. The sigmoid function is defined as 
+  1 divided by (1 + exp(-x)). It's curve is like an "S" and is like a smoothed 
+  version of the Heaviside function. For small values (<-5) the sigmoid returns 
+  a value close to zero and for larger values (>5) the result of the funtion gets
```
Comment (kyscg) [inline tensorflow/python/keras/activations.py]: function*

**Review by kyscg — CHANGES_REQUESTED** [review]
Try these @aweers

**Comment by aweers** [comment]
Thanks for the feedback @kyscg! I updated the descriptions with your suggestions.

**Review by kyscg — DISMISSED** [review]
LGTM @aweers 
Cheers!!

File: tensorflow/python/keras/activations.py [hunk 6]
```diff
@@ -42,14 +42,18 @@
 
 @keras_export('keras.activations.softmax')
 def softmax(x, axis=-1):
-  """Softmax activation function.
+  """The softmax activation function transforms the outputs so that all values are in 
+  range (0, 1) and sum to 1. It is often used as the activation for the last layer of a 
+  classification network because the result could be interpreted as a probability 
+  distribution. To calculate the softmax, all inputs `x` become exp(x) divided by the sum
```
Comment (alextp) [inline tensorflow/python/keras/activations.py]: I think an equation reads better than the textual description of it here (the sentence starting with "to calculate" should just show exp(x)/ tf.reduce_sum(exp(x))

**Review by alextp — CHANGES_REQUESTED** [review]

**Review by alextp — DISMISSED** [review]

**Comment by rthadur** [comment]
@aweers please resolve conflicts

**Comment by aweers** [comment]
Hope, everything is ok now! ;)

**Comment by rthadur** [comment]
> Hope, everything is ok now! ;)

@alextp can you please review

File: tensorflow/python/keras/activations.py [hunk 7]
```diff
@@ -179,6 +184,17 @@ def tanh(x):
...
+  Applies the sigmoid activation function. The sigmoid function is defined as 
+  1 divided by (1 + exp(-x)). It's curve is like an "S" and is like a smoothed 
+  version of the Heaviside (Unit Step Function) function. For small values 
+  (<-5) the sigmoid returns a value close to zero and for larger values (>5) 
+  the result of the function gets close to 1.
+  Arguments:
+      x: A tensor or variable.
+  Returns:
+      A tensor.
+=======
```
Comment (alextp) [inline tensorflow/python/keras/activations.py]: Remove =====

**Review by alextp — DISMISSED** [review]

**Comment by aweers** [comment]
How to fix this error? Can't even see any details when clicking on 'Details'

File: tensorflow/python/keras/activations.py [hunk 8]
```diff
@@ -179,6 +184,16 @@ def tanh(x):
...
+  Applies the sigmoid activation function. The sigmoid function is defined as 
+  1 divided by (1 + exp(-x)). It's curve is like an "S" and is like a smoothed 
+  version of the Heaviside (Unit Step Function) function. For small values 
+  (<-5) the sigmoid returns a value close to zero and for larger values (>5) 
+  the result of the function gets close to 1.
+  Arguments:
+      x: A tensor or variable.
+  Returns:
+      A tensor.
   """Sigmoid activation function.
```
Comment (alextp) [inline tensorflow/python/keras/activations.py]: This extra quote is a syntax error

**Comment by aweers** [comment]
Thanks @alextp, I hope now everything is right... ;)

**Review by alextp — APPROVED** [review]
